compiler/stable_mir/src/mir/body.rs

```diff
@@ -913,10 +913,16 @@ pub enum Mutability {
     Mut,
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum Unsafe {
+    Yes,
+    No,
+}
+
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
 pub enum Safety {
     Unsafe,
-    Normal,
+    Default,
```

Comment:
Don't we already know if a function / static is safe or unsafe by the time we are translating to StableMIR? If so, wouldn't it be better to keep Unsafe / Safe (Normal)?

I'm assuming that the way this is designed today, the `Default` value will require users to perform an extra query to know the context where this Fn / Static was declared to know whether it is safe or not.

Reply:
BTW, this is just a suggestion. I know this is a big change already. If it makes sense, maybe just add a fixme comment for us to improve this later. Thanks